app/src/main/res/layout/activity_help_legend.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I spaced it out a little more. If we do not like it we can change it. I USED A STRING FOR THE PADDING BETWEEN THEM SO IT IS EASY TO CHANGE!!